engine/execution/ingestion/engine.go

```diff
@@ -683,11 +681,7 @@ func (e *Engine) addCollectionToMempool(
 		}
 
 		// record collection max height metrics
```

Comment:
comment might be redundant now, since the code already plainly states the same thing.